tomography_preprocessing/tilt_series_alignment/aretomo/_utils.py

```diff
@@ -10,20 +10,16 @@
 def get_tilt_series_alignment_parameters(
```

Comment:
can you refactor this to `get_tilt_series_alignment_data` so that it returns `xf` and `tlt` only, then the specimen shifts are calculated later from those (outside this function) wherever they're needed?